engines/supernova/detection.cpp

```diff
@@ -60,13 +60,22 @@ static const ADGameDescription gameDescriptions[] = {
 	// Mission Supernova 2
 	{
 		"msn2",
-		NULL,
+		nullptr,
 		AD_ENTRY1s("ms2_data.000", "e595610cba4a6d24a763e428d05cc83f", 24805),
 		Common::DE_DEU,
 		Common::kPlatformDOS,
 		ADGF_UNSTABLE,
 		GUIO1(GUIO_NONE)
 	},
+	{
+		"msn2",
```

Comment:
why this duplicate entry is here?

Reply:
huh.. Guess there went something wrong when I resolved a conflict.